spec/u-namespace.md

```diff
@@ -0,0 +1,72 @@
+# MessageFormat 2.0 Unicode Namespace
+
+The `u:` namespace is reserved for use by the Unicode Consortium.
```

Comment:
Here's where the definition goes, I guess? Perhaps:

```suggestion
The `u:` _namespace_ is reserved for the definition of _options_
for use in assisting the translation process 
or which affect the _function context_ of the specific _expressions_
in which they appear.
The CLDR Technical Committee of the Unicode Consortium
manages the specification for this namespace, hence the name `u:`.
```

Reply:
> for use in assisting the translation process

Isn't this handled by attributes?

Reply:
Leaving that out:
```suggestion
The `u:` _namespace_ is reserved for the definition of _options_
which affect the _function context_ of the specific _expressions_
in which they appear.

The CLDR Technical Committee of the Unicode Consortium
manages the specification for this namespace, hence the name `u:`.
```

This doesn't cover `u:id` on markup, though. Drawing a bit of a blank on exactly how to include that.